### DISCUSSION TITLE
¿Comó subir katas?

### DISCUSSION THREAD

**Discussion by KarlaILM** [LaunchX-InnovaccionVirtual/CursoIntroPython#102] [Q&A]
Hola compañeros, tengo una duda, como es que demos subir las katas

**Comment by doguedogue** [comment]
crea un repositorio en github y sube tus archivos con los ejercicios de las katas

**Comment by OscarPedroza** [comment]
Se van a subir a un repositorio de github, en la reunión de mañana se abordará mas ese tema.

**Comment by AzaelBarbosa** [comment]
Crearas un repositorio con tus Katas, despues el dia de manana publicaran un Forms, donde pondras tu nombre y el link a ese repositorio con las Katas

**Comment by D-a-v-i-d-Vargas** [comment]
Hola Buen día, yo tenía la misma duda, pero para crear tu repositorio y que puedas ver como se hace, paso a paso ve a el siguiente link: 

https://teams.microsoft.com/_#/tab::17010906-4fdd-46ad-b7bb-7224f2162d2c/General?threadId=19:4SY7l6F1oATeZRTW_mEV7y0PHO9MXkf7aHaJh4PRZXI1@thread.tacv2&ctx=channel

En la hora 2 minuto 56 (adelantas el video haste llegar allí), nuestra comander explica a detalle como hacer tu repositorio para subir tus katas. 

Espero te sirva la info, lo puse demaciado explicado quiza, porque luego uno se pierde un poco. Mucho exito, saludos.

P.d. yo también voy empezando desde 0 :)
